src/OrbitGl/SamplingReport.cpp

```diff
@@ -47,6 +51,25 @@ void SamplingReport::ClearReport() {
   }
 }
 
+const orbit_client_data::CallstackData* SamplingReport::GetCallstackData() const {
+  return callstack_data_;
+}
+
+[[nodiscard]] std::optional<absl::flat_hash_set<uint64_t>> SamplingReport::GetSelectedCallstackIds()
```

Comment:
Could this function be tested?

Reply:
As promised, tests will be added in the next PR.